backend/lobby/consumers.py

```diff
@@ -79,15 +81,21 @@ def get_or_create_user(self, username, room):
 
     def get_or_create_game(self, username, room):
         game, created = GameState.objects.get_or_create(room_name=room)
-
         i_repository_game = IRepositoryGame()
+        i_repository_player = IRepositoryPlayer()
 
         if not i_repository_game.get_game_by_room(room):
             game_store = i_repository_game.save_game(room)
             print("Room created with id {}".format(game_store))
         else:
             error = 'Room already exist, Unable to create Game with room: ' + room
 
+        if not i_repository_player.get_players_by_username_and_room(username, room):
+            player_store = i_repository_player.save_player(username, room)
+            print("Player created with id {}".format(player_store))
+        else:
+            error = 'Player in this room already exist, Unable to create Player in room: ' + room
```

Comment:
This isn't so much that the player can't be created, it's that the player can't be added to the database because a database entry exists. Since this database is used for analytics and not gameplay, this error message seems a little misleading. And do we want to tell the player that the background analytics aren't working? would they care?

Reply:
Great point, updated the error message to clarify